addons/project/static/src/views/fields/project_task_priority_switch_field.js

```diff
@@ -0,0 +1,36 @@
+/** @odoo-module **/
+
+import { sprintf } from "@web/core/utils/strings";
+import { registry } from "@web/core/registry";
+import { PriorityField } from "@web/views/fields/priority/priority_field";
+import { useService } from "@web/core/utils/hooks";
+import { useState, useEffect } from "@odoo/owl";
+
+export class PrioritySwitchField extends PriorityField {
+    setup() {
+        this.state = useState({
+            index: -1,
+        });
+        if (this.props.record.activeFields[this.props.name].viewType !== "form") {
+            return;
+        }
+
+        const commandService = useService("command");
+        useEffect(
+            (currentValue) => {
+                if (!currentValue) {
+                    return;
+                }
+                const [id, name] = this.options.find((option) => option[0] !== currentValue);
+                return commandService.add(
+                    sprintf(this.env._t("Set priority as %s"), name),
+                    () => this.props.update(id),
+                    { category: "smart_action", hotkey: "alt+r" }
+                );
+            },
+            () => [this.props.value]
+        );
+    }
+}
```

Comment:
As we discussed a few days ago, why not use this commit to manage the display or not of a command? https://github.com/odoo/odoo/pull/104830